plans/create.pp

```diff
@@ -65,10 +72,38 @@
   Optional[Integer[1, 10]] $node_count           = 1,
   Optional[Hash[String[1], String[1]]] $tags     = {},
   Optional[Hash[String[1], String[1]]] $provider_options = undef,
+
+  ##GCP Specific Parameters
+  Optional[String[1]] $project                    = undef,
```

Comment:
I think network right now can be used with subnet and remaining params `subnetwork` & `subnetwork_project` can be make part of provider_options ?

Reply:
Subnetwork, we can make part of provider_options. But subnetwork_project is by default is same as project and user can intentionally make subnetwork_project as well other than project. In such a scenario, we need to update the variable which wont be possible in hash if we initially make it a part of provider_options hash. Thoughts @Ramesh7 ??